<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QWidget" name="mainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f8fa;
font: 11pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QWidget" name="widget_mainMenu" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>640</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	color: #000000;
	background-color: #fafafa;
	border-width: 5px;
	border-color: #fafafa;
	border-bottom-color: #c8c8c8;
	border-style: solid;
	border-radius: 0px;
}

QPushButton:disabled 
{
	color: #0a0a0a;
	background-color: #f0f0f0;
	border-width: 5px;
	border-color: #f0f0f0;
	border-bottom-color: #c8c8c8;
}

QPushButton:hover
{
	background-color:#f5f8fa ;
	border-color: #f5f8fa;
	border-bottom-color: #c8c8c8;
}
QPushButton:focus
 {
	background-color: #ebf0f5;
	border-color: #ebf0f5;
	border-bottom-color: #c8c8c8;
}

QPushButton:pressed 
{
	background-color: #ebf0f5;
	border-color: #ebf0f5;
	border-bottom-color: #c8c8c8;
}

QPushButton:checked {
	background-color: #ebf0f5;
	border-color: #ebf0f5;
	border-bottom-color: #0064c8;
}</string>
   </property>
   <widget class="QPushButton" name="btn_mainMenu_load">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>载入</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mainMenu_list">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>待选列表</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mainMenu_workList">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>当前任务</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mainMenu_settings">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="text_mainTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>LUD - Netease Cloud Music Downloader</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_state">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ebf0f5;
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>准备就绪</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_workWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>640</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LUD.qrc"/>
 </resources>
 <connections/>
</ui>
